find_xml_file/find_xml_file.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+
+def find_xml_file(folder):
+    files = os.listdir(folder)
+
+    xml_file = None
```

Comment:
This might need to be renamed xml_files if we are going to possibly grab more than one.